### PR TITLE
Changelog1.2.6

### PR DESCRIPTION
**Pull request by paulhainsworth-chia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,24 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Changed
+
+- Updated to BLS 1.0.6.
+- Updates to the Rust conditions parser.
+- Adjusted plot refresh parameter to improve plot loading times.
+
+### Fixed
+
+- Fixed memory leak in versions 1.2.4 and 1.2.5 related to how the node handles compact VDFs generated from blueboxes.
```
Comment (emlowe) [inline CHANGELOG.md:19]: All versions - not just 1.2.4/1.2.5

Comment (emlowe) [reply CHANGELOG.md:19]: Also missing

- Fixed issues with reloading plot files detected as bad (this can happen during plot copying)

Comment (paulhainsworth-chia) [reply CHANGELOG.md:19]: Thx will fix @emlowe

**Review by emlowe — DISMISSED** [review]
Looks good

**Review by jack60612 — DISMISSED** [review]
Looks ready for release with the merge from main!

**Comment by jack60612** [comment]
If possible getting this included would be nice: https://github.com/Chia-Network/chia-blockchain-gui/pull/531

**Comment by hoffmang9** [comment]
This release is intentionally only fixes and not enhancements per-se.

**Pull request merged**